objectivec/DevTools/pddm.py

```diff
@@ -482,13 +482,14 @@ def lines(self):
         if self._macro_collection:
           # Always add a blank line, seems to read better. (If need be, add an
           # option to the EXPAND to indicate if this should be done.)
-          result.extend([_GENERATED_CODE_LINE, ''])
+          result.extend([_GENERATED_CODE_LINE, '// clang-format off', ''])
           macro = line[directive_len:].strip()
           try:
             expand_result = self._macro_collection.Expand(macro)
             # Since expansions are line oriented, strip trailing whitespace
             # from the lines.
             lines = [x.rstrip() for x in expand_result.split('\n')]
+            lines.extend(['// clang-format on', ''])
```

Comment:
The blank line being inserted here seems like a negative, is there a reason you added it?

Reply:
Similar to the blank line in line 483 it just seemed to read better. Happy to remove if you prefer.

Reply:
Go ahead and drop it.  It current sticks out with a blank before after, vs. merging with the block like what happens in that other case.

Reply:
done